ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -69,6 +69,15 @@ open class CommandBarItem: NSObject {
         }
     }
 
+    /// Image displayed behind the `iconImage` to provide fill or accent if desired
+    @objc public var accentImage: UIImage? {
```

Comment:
curious, should we be more conservative that even if user passes in default rendering image, we will treat it like templated image?

Reply:
In this case, I think it's best to not override the renderingMode and let the provider of the `CommandBarItem`s determine the rendering mode. If the icon is tinted with a specific color and with `original` set as template before being set to the `CommandBarItem.accentImage`, changing the renderingMode to `template` would cause the color disappear and the icon would respect the system theme color which is not what the user intended.

Reply:
If we wanted to treat all images as templates, I think we would need to expose an additional `accentImageTintColor` property which could be used to tint the images on `UIUserInterfaceStyle` changes.

Reply:
After some more thinking, I think exposing the accentImageTintColor property to tint the image and forcing the accentImage to use `alwaysTemplate` does remove some ambiguity from how the component works. I've made the necessary changes in the latest revision.